surefire-booter/pom.xml

```diff
@@ -115,11 +115,12 @@
       </plugin>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M3</version>
```

Comment:
why are we using this older version "3.0.0-M3" ? what's the reasoning about choosing this one ? could it be 2.x ?

Reply:
I can try with 2.22.2. Let's see what happens...

Reply:
@eolivelli 
I have used the 2.22.2 but it gives me bad result
`[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0`
Sorry!